pythonforandroid/archs.py

```diff
@@ -30,13 +30,24 @@ def include_dirs(self):
                 d.format(arch=self))
             for d in self.ctx.include_dirs]
 
-    def get_env(self, with_flags_in_cc=True):
+    def get_env(self, with_flags_in_cc=True, clang=False):
```

Comment:
So basically that file change mainly makes it possible to compile with `clang`.
That's very interesting and probably deserves a PR on its own to make it faster to the tree